src/test/java/hw5/pages/DifferentElementsPage.java

```diff
@@ -0,0 +1,148 @@
+package hw5.pages;
+
+import hw5.beans.Color;
+import hw5.pages.components.HeaderMenu;
+import hw5.pages.components.LeftSideMenu;
+import hw5.pages.components.RightSideBar;
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+
+public class DifferentElementsPage {
+
+    private WebDriver driver;
+
+    private LeftSideMenu leftSideMenu;
```

Comment:
You've already had these fields in the `BaseElements` class. Why do you decide do this code duplication?

Reply:
my fault